util/topgen/templates/toplevel.sv.tpl

```diff
@@ -273,6 +273,12 @@ module top_${top["name"]} #(
   assign rv_core_ibex_boot_addr = ADDR_SPACE_ROM;
 % endif
 
+  ## Not all top levels have a lifecycle controller.
+  ## For those that do not, always enable ibex.
+% if not lib.is_lc_ctrl(top["module"]):
+  assign rv_core_ibex_lc_cpu_en = lc_ctrl_pkg::On;
+% endif
+
```

Comment:
@msfschaffner and @tjaychen : I've followed the ROM example here but could have also just defined the `On` as default inside `rv_core_ibex.hjson`. However, I thought this is probably safer as it doesn't mask the case where in a design with LC controller we accidentally forgot to connect the signal (then the `On` default would be used).

Reply:
I am fine with this templating approach to put us on the safe side of things.

Reply:
Alright. Thanks for having a look Michael!